### PR TITLE
Frontend RTL

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

File: frontends/web/src/components/transactions/transaction.css [hunk 1]
```diff
@@ -152,6 +169,13 @@
     transition: all 0.1s ease;
 }
 
+[lang="ar"] .toggleContainer,
```
Comment (benma) [inline frontends/web/src/components/transactions/transaction.css]: why not `.rtl .toggleContainer`?

Comment (thisconnect) [reply frontends/web/src/components/transactions/transaction.css]: doesn't work in CSS Modules, but maybe could be enabled somehow?

Comment (benma) [reply frontends/web/src/components/transactions/transaction.css]: Then please use css modules :) e.g. set `className={isRTL() ? style.rtl : null}` in an appropriate parent element, then you can use `.rtl .toggleContainer` etc. in the css modules.

Comment (KasparEtter) [reply frontends/web/src/components/transactions/transaction.css]: If `.rtl` is defined globally, you can probably use https://github.com/css-modules/css-modules#exceptions.

Comment (thisconnect) [reply frontends/web/src/components/transactions/transaction.css:172]: Picking this up soon™ `:global(.rtl) .toggleContainer` should work, thanks for the hints.

Comment (thisconnect) [reply frontends/web/src/components/transactions/transaction.css:172]: Looking also into https://developer.mozilla.org/en-US/docs/Web/CSS/:dir

**Comment by thisconnect** [comment]
Sorry for keeping this open so long, will revisit later

**Pull request closed without merging**